api/api.js

```diff
@@ -775,7 +776,12 @@ async function fetchValidatorAvatars(keybaseUsernames) {
   const avatarMap = new Map();
   let batch = keybaseUsernames.splice(0, MAX_KEYBASE_USER_LOOKUP);
   while (batch.length > 0) {
-    const usernames = batch.join(',');
+    // see: https://github.com/keybase/keybase-issues/issues/757#issuecomment-45229307
+    const usernames = flow(
+      uniq,
+      filter(x => x.match(/^[a-zA-Z0-9][a-zA-Z0-9_]{1,14}$/)),
```

Comment:
Nice regex! For completeness, we might as well filter out repeated underscores too

Reply:
👍 awesome idea, will implement!

Reply:
done!